### ISSUE TITLE
server readiness probe doesn't support disabling http (port 8500)

### ISSUE THREAD

**Issue by infa-bsurber** [hashicorp/consul-helm#157]
As per the TLS bootstrapping guide here: https://learn.hashicorp.com/consul/security-networking/certificates#step-1-setup-consul-servers-with-certificates

It advises disabling http once https is available via this configuration:
```
{
  "verify_incoming": true,
  "verify_outgoing": true,
  "verify_server_hostname": true,
  "ca_file": "consul-agent-ca.pem",
  "cert_file": "dc1-server-consul-0.pem",
  "key_file": "dc1-server-consul-0-key.pem",
  "ports": {
    "http": -1,
    "https": 8501
  }
}
```

With current consul-helm, the server's readiness probe is hardcoded to use http on port 8500 so there's no way to:
a) probe that my consul's https configuration is working + ready
b) stop probing 8500

**Comment by adilyse** [comment]
Hi @infa-bsurber,

This is one of several issues with the current chart that make TLS setup difficult using the Helm chart. We are currently taking a comprehensive look at how to support TLS more effectively through consul-k8s and the Helm chart, where this will be addressed.

**Comment by infa-bsurber** [comment]
Do you have a consul-k8s issue for me to follow @adilyse  ?

I've had some success enabling TLS via extraVolumes & extraConfig, so I'm curious what the additional work on supporting it looks like.

**Comment by adilyse** [comment]
The comprehensive effort is being tracked internally, so there's not a specific github issue associated with it. You will start to see some PRs show up here and in the `consul-k8s` repo related to it soon.

**Comment by infa-bsurber** [comment]
Great, thanks for the info

**Comment by ishustava** [comment]
Hey @infa-bsurber, we've shipped TLS support in version [0.16.0](https://github.com/hashicorp/consul-helm/issues/157).

I'm going to close this issue, but if the latest release doesn't work for you, please let us know and we'll re-open this issue. Thanks!
